### PR TITLE
refactor(face-db): have code consistency similar to other exam…

### PR DESCRIPTION
**Pull request by kaushikb11**
The motivation behind this PR is when a User is trying out multiple examples, code consistency & similar patterns could be helpful. Refactored some minor changes in the `face-db-search` example. README seems to be getting updated in #158.

### CODE REVIEW

File: face-db-search/app.py [hunk 1]
```diff
@@ -3,63 +3,43 @@
...
-def dryrun():
-    f = Flow.load_config('flow-query.yml')
-
-    with f:
-        pass
+@click.command()
+@click.option('--task', '-t')
+@click.option('--num_docs', '-n', default=500)
+def main(task, num_docs):
+    config()
```
Comment (JoanFM) [inline face-db-search/app.py:23]: would make more sense to have task as param of the config then?

File: face-db-search/app.py [hunk 2]
```diff
@@ -3,63 +3,45 @@
...
 from jina.flow import Flow
 
-num_docs = os.environ.get('MAX_DOCS', 500)
-image_src = '/tmp/jina/celeb/lfw/**/*.jpg'
 
-
-def config():
-    parallel = 2 if sys.argv[1] == 'index' else 1
-    shards = 8
+def config(task):
```
Comment (nan-wang) [inline face-db-search/app.py:11]: `task` is not used.

**Review by nan-wang — CHANGES_REQUESTED** [review]

File: face-db-search/app.py (hunk 2 above)
Comment (kaushikb11) [reply face-db-search/app.py:11]: @nan-wang `parallel = 2 if task == 'index' else 1` There's this condition in the `config` function to set `PARALLEL` env variable.

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**